.github/workflows/release-build-check.yml

```diff
@@ -70,7 +70,6 @@ jobs:
 
         # Since the release build check is a scheduled job, a failure won't be shown on any
         # PR status. To notify the team, we send a message to our Slack channel on failure.
-      - uses: actions/checkout@v3 # checks out the latest slack action
```

Comment:
This checkout action has been removed because the post-run actions of the `check-build` action crash with out-of index if the number of steps in that action has changed since the last release.